static/css/components/buttonCta.less

```diff
@@ -6,26 +6,36 @@
 // override default anchor styles
 a.cta-btn {
   font-size: 14px;
-  color: @white !important;
+  color: @white;
   width: 100%;
   display: block;
   border: 0;
   border-radius: 5px;
   box-sizing: border-box;
   cursor: pointer;
-  text-decoration: none;
   font-family: @lucida_sans_serif-1;
   text-align: center;
   padding: 7px 20px;
   white-space: nowrap;
   background-color: @mid-grey;
   line-height: 1.5em;
+  text-decoration: none;
+
+  &:link, &:visited {
+    color: @white;
+    text-decoration: none;
+  }
   &--available {
     background-color: @primary-blue;
   }
   &--unavailable {
     background-color: @orange;
   }
+  &--preview, &--preview:link, &--preview:visited {
```

Comment:
Do we really need to add another button to our long list? Can we not use one of the existing ones?

Reply:
One option is to have a generic secondary call to action which can be used both for the preview and the sponsorship button.

Reply:
punting for now, will not fix